cypress/integration/datetime.js

```diff
@@ -0,0 +1,192 @@
+/* Test date, time and datetime formats.
```

Comment:
I appreciate that this is an extensive test suite for all things datetime, but it increases the test time by almost 3-4 minutes. Let's reduce this to a few nominal tests. On the UI side we should only test the user input and ensure that the values are saved properly in the form. You already have covered a lot of formats in Python unittest.

Reply:
Reduced to testing one date format and two time formats.

Reply:
Reduced to testing a single date format and both time formats.